Essentials/src/com/earth2me/essentials/UserData.java

```diff
@@ -898,14 +898,14 @@ public void setAcceptingPay(boolean acceptingPay) {
         save();
     }
 
-    private boolean confirmPay = false; // players deny pay confirmation by default
+    private Boolean confirmPay;
 
-    public boolean _getConfirmPay() {
-        return config.getBoolean("confirm-pay", false);
+    private Boolean _getConfirmPay() {
+        return (Boolean) config.get("confirm-pay");
```

Comment:
Why not use the default from `default-disabled-confirm-commands` here? It makes the `isPromptingPayConfirm` and `isPromptingClearConfirm` methods messy, and means we can just use the primitive `boolean`.

Reply:
Server reload. That config variable may change.

Reply:
I don't see how that affects using it as the default in case the user config doesn't yet have that set. As far as I can see, that would be equivalent to what the PR does already, just much tidier and without having the config value retrieval itself split across two methods for each confirmation.

Reply:
Any thoughts on this supa?